docs/02_cleos/03_command-reference/set/set-action-permission.md

```diff
@@ -64,7 +65,7 @@ executed transaction: 4eb4cf3aea232d46e0e949bc273c3f0575be5bdba7b61851ab51d927cf
 #         eosio <= eosio::linkauth              {"account":"bob","code":"scontract1","type":"hi","requirement":"customp1"}
 ```
 
-2. permissions on a _bye_ action deployed to the _scontract1_ account so that the _bob_ account's `active` permission and _customp2_ permission are authorized:
+2. Set permissions on a _bye_ action deployed to the _scontract1_ account so that the _bob_ account's `active` permission and _customp2_ permission are authorized:
```

Comment:
I still don't understand why `active` and _customp2_ have to use different formatting styles. Both are permissions and they are both instantiated from the same class.

Reply:
The idea is to differentiate between defaut and custom permissions to emphsise this for this how to